src/screens/Settings/components/LanguageScreen/components/LanguageItem/LanguageItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Image, Pressable, Text, View } from 'react-native';
+
+import { Icons } from '@UI/Icons';
+
+import { styles } from './styles';
+
+interface Props {
+  icon: keyof typeof Icons;
+  language: 'en' | 'ua' | undefined;
+  onPress: () => any;
+  title: string;
+}
+
+export function LanguageItem({ icon, language, title, onPress }: Props): JSX.Element {
+
+  return (
+    <Pressable style={styles.item} onPress={onPress}>
+      <View style={styles.itemBackground}>
+        <Image source={Icons[icon]} style={styles.buttonImage} />
+        {language === icon && (
```

Comment:
how can language be equal to icon? Either make proper naming or refactor this logic